scripts/generate_mkdocs.py

```diff
@@ -38,6 +38,35 @@ class Section(NamedTuple):
 )
 
 
+def clean_file_content(content: str, title: str) -> str:
+    """Add missing title, fix the headers depth, and remove trailing empty lines."""
+    lines = content.splitlines()
+    if lines[0].startswith("# "):
+        return content
```

Comment:
What's this condition here?

Reply:
This was used for the acknowledgment section.

Reply:
Which is not included anymore in the documentation.
Should it be added back?

Reply:
Oh, I think it's fine for that to exist in the README but not the docs. Same with "Who's Using Ruff".

Reply:
I thought you mentioned the `if not lines[0].startswith("# "):` that was removed in the last commit.
This one is used to avoid cleaning `Overview` which is already without error and does not need to change the header's depth.